pydbtools/utils.py

```diff
@@ -1,25 +1,40 @@
-from typing import Tuple, Optional
+import inspect
 import os
 import re
-import sqlparse
-from urllib.parse import urlparse, urljoin, urlunparse
-import sql_metadata
-import inspect
-import boto3
-from botocore.credentials import (
-    InstanceMetadataProvider,
-    InstanceMetadataFetcher,
-)
 from functools import reduce
-import awswrangler as wr
+from typing import Optional, Tuple
+from urllib.parse import urljoin, urlparse, urlunparse
 
+import awswrangler as wr
+import boto3
+import sql_metadata
+import sqlparse
+from botocore.credentials import InstanceMetadataFetcher, InstanceMetadataProvider
 
 # Set pydbtool params - if you were so inclined to change them
 bucket = "mojap-athena-query-dump"
 temp_database_name_prefix = "mojap_de_temp_"
 aws_default_region = os.getenv(
     "AWS_DEFAULT_REGION", os.getenv("AWS_REGION", "eu-west-1")
 )
+aws_role_regex_rules = [
+    (
+        r"@digital.justice.gov.uk$",  # digital justice
+        r"@digital.justice.gov.uk",
+    ),
+    (
+        r"^[0-9]+",  # numeric
+        None,
+    ),
+    (
+        r"alpha_user_",  # alpha user
+        "alpha_user_",
```

Comment:
I'm not sure we necessarily want to remove the alpha_user part, I think it's useful to be able to see which databases are created by AP users, and would be consistent with [Rdbtools](https://github.com/moj-analytical-services/Rdbtools/blob/main/R/utils.R)